src/libs/logos.js

```diff
@@ -13,6 +13,7 @@ const brandLogoMaker = (size, color = false) =>
     style: { height: size, marginRight: '1.5rem' },
   });
 
+// Needs to be capitalized to be a TSX Component
 export const registrationLogo = () =>
```

Comment:
This isn't used anywhere else, so can we rename it instead of creating an alias?

```suggestion
export const RegistrationLogo = () =>
```